### PR TITLE
Add a paginated view of all comments

### PR DESCRIPTION
**Pull request by smarzban**
Each article that has more than the max number of comments should have a ‘View more comments’ at the bottom of the page that links to a paginated view of all comments.

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony @bruskiza ready for review :)

**Comment by bruskiza** [comment]
beyond me i'm afraid.
#lacking_python_fu

File: ndohyep/templates/comments/report_response.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <br>
+  <h4 class="left">{% trans "Thanks!" %}</h4>
+</div>
+<div class="block">
+  <br>
+  <p><strong>{% trans "This comment has been reported." %}</strong>
+  {% trans "It will be reviewed by the B-Wise team and may be removed if it breaks our Platform rules" %}</p>
+</div>
+<div class="title {{article.get_parent_section.get_effective_extra_style_hints}}">
+  <a href="javascript:history.back();" class="button">{% trans "Back" %}</a>
```
Comment (smn) [inline ndohyep/templates/comments/report_response.html:15]: This javascript will likely break in many situations. Can we not just pass along the link (or even article-id) of the original article and create a regular `href` link?

**Comment by smn** [comment]
:+1:, reluctantly, we cannot deliver the current scope with out it easily. We need to be on the lookout for this in future scopes though. A backup mechanism would be to use the request.session to keep track of where we came from.

**Pull request merged**